docs/faq-ps4-ps5-compatibility.md

```diff
@@ -50,3 +50,23 @@ If you're using an OLED display, the input mode will change from `PS4` to `PS4:A
 The GP2040-CE project will not provide ***any*** files or information related to acquiring these keys and files.
 
 !> **Do not ask via any of communication channels (e.g. social media, direct messages, Github Issues, GP2040-CE) as this will result in a permanent blacklist/ban.**
+
+## What controller or dongle do I use for PS Passthrough?
+
+This depends on whether you are aiming to avoid the 8 minute timeout on PS4 or are looking to get your controller working on PS5.
+
+### Playstation 4
+
+For avoiding the 8 minute timeout on PS4, any PS4 controller or converter should work unless the device is not a valid and accepted as a proper controller on its own with a PS4.
+
+### Playstation 5
+
+For PS5 compatibility, there are a number of devices that the PS5 recognizes as a "specialty" controller (e.g. arcade stick, racing wheel, flight sim joystick, etc.) where the developer of the game has supported the use of "specialty" controllers. Any device works as a specialty PS4 controller (arcade stick/racing wheel/flight sim joystick) or a dongle that allows a controller to authenticates as such should work for games that support it. Note that this is not all games, but most fighting games and vehicle simulators do.
+
+Some devices that do work for PS5 include, but are not limited to;
+
+- [MagicBoots FPS Adapter Joysick Converter for PS4 v1.1](https://www.mayflash.com/product/MAGPS4.html)
+- [N5 for PXN Game Steering Wheel](https://pxn-game.com/products/n5-for-pxn-game-steering-wheel)
+- PCBs from Licensed PS4 Arcade Sticks
+
+Because your experience and purchased products may vary, GP2040-CE will not bear the responsibility of providing specific recommendations beyond these known working devices.
```

Comment:
```suggestion
Because your experience and purchased products may vary, GP2040-CE will not provide specific recommendations beyond these known working devices.
```